### PR TITLE
Bugfix/schedule report layout bug

### PR DESCRIPTION
**Pull request by ltwheeler**
Adds enrollment data to schedule summary report, if its available.
Fixes a bug in the layout caused by missing enrollment data/column.

Issue:
https://trello.com/c/IiUJfpYC/1547-schedulesummaryreport-columns-do-not-end-correctly-when-location-is-set

### CODE REVIEW

File: app/shared/services/dwService.js [hunk 1]
```diff
@@ -37,6 +37,33 @@ angular.module('sharedApp')
...
+			/**
+			 * Provides a list of census snapshots based on the parameters given.
+			 * SubjectCode param is required.
+			 */
+			getDwCensusData: function(subjectCode, courseNumber, termCode) {
+				if (subjectCode == null) { return null; }
+
+				var params = "?subjectCode=" + subjectCode;
+
+				if (courseNumber) {
```
Comment (cthielen) [inline app/shared/services/dwService.js]: Array.join() is a cleaner way to handle the number of args.

File: app/shared/services/dwService.js [hunk 2]
```diff
@@ -37,6 +37,33 @@ angular.module('sharedApp')
...
+
+				if (termCode) {
+					if (params.length > 0) {
+						params += "&";
+					}
+
+					params += "termCode=" + termCode;
+				}
+
+				return apiService.get("/census" + params + "&token=" + dwToken, null, dwUrl);
```
Comment (cthielen) [inline app/shared/services/dwService.js]: I must be missing something. Where does the value of dwToken get filled in?

Comment (ltwheeler) [reply app/shared/services/dwService.js]: dwToken, dwUrl and serverRoot come from clientConfig.js, and are globally accessible by including them alongside the rest of the scripts for each app.

Comment (cthielen) [reply app/shared/services/dwService.js]: Thanks!

**Pull request merged**